src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -213,7 +217,6 @@ public static int execute(final String[] args) throws Exception {
         options.addOption(Option.builder("r").longOpt("retrieve").desc("retrieve current config").build());
         options.addOption(Option.builder("f").longOpt("file").hasArg().argName("file").desc("file").build());
         options.addOption(Option.builder("t").longOpt("type").hasArg().argName("file-type").desc("file-type").build());
-        options.addOption(Option.builder("tsalias").longOpt("truststore-alias").hasArg().argName("alias").desc("Truststore alias").build());
```

Comment:
Why removing this?

Reply:
Should the ports at line#212 and line#417 and line#419 be changed?
```
options.addOption(Option.builder("p").longOpt("port").hasArg().argName("port").desc("OpenSearch transport port (default: 9300)").build());
```
```
        if(port < 9300) {
            System.out.println("WARNING: Seems you want connect to the OpenSearch HTTP port."+System.lineSeparator()
                             + "         securityadmin connects on the transport port which is normally 9300.");
        }
```

Reply:
done